website/content/docs/k8s/multiport/configure.mdx

```diff
@@ -0,0 +1,478 @@
+---
+layout: docs
+page_title: Configure multi-port services 
+description: Learn how to enable the v2 catalog and configure services to support multiple ports in Consul on Kubernetes. You can configure multiple ports on a single service or multiple services and ports in a single container.
+---
+
+# Configure multi-port services
+
+<Warning>
+
+Multi-port services are part of a beta release. This documentation supports testing and development scenarios. Do not use multi-port services or the v2 catalog API in secure production environments.
+
+</Warning>
+
+This page describes the process for integrating a service that uses multiple ports in a single container when running Consul on Kubernetes deployments. It includes example configurations to demonstrate an end-to-end deployment test of Consul's multi-port features.
+
+## Prerequisites
+
+Registering multi-port services with Consul requires Kubernetes. Multi-port services are not supported on VM deployments.
+
+There are also software version prerequisites:
+
+- `consul` v1.17.0
+- `consul-k8s` CLI v1.3.0+ or `hashicorp/consul` Helm chart release v1.3.0+
+
```

Comment:
```suggestion
The following software versions are required:

- Consul v1.17.0
- `consul-k8s` CLI v1.3.0+ or `hashicorp/consul` Helm chart release v1.3.0+

```
At least for core Consul, I think we normally refer to the product name in requirements, i.e., "Consul v1.17.0". We've been less consistent with the K8s binary, though. Was your thinking that if we refer to the binary for one requirement in a topic that we should refer to the binary for all requirements in the the topic?